### PR TITLE
Added buffer handler config parameter

### PR DESCRIPTION
**Pull request by Matrooskin**

### CODE REVIEW

**Comment by xabbuh** [comment]
You need to update the XML schema too.

**Comment by Matrooskin** [comment]
Updated

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -318,6 +319,7 @@ public function getConfigTreeBuilder()
                             ->scalarNode('min_level')->defaultValue('DEBUG')->end() // filter
                             ->scalarNode('max_level')->defaultValue('EMERGENCY')->end() //filter
                             ->scalarNode('buffer_size')->defaultValue(0)->end() // fingers_crossed and buffer
+                            ->scalarNode('flush_on_overflow')->defaultValue(false)->end() // buffer
```
Comment (stof) [inline DependencyInjection/Configuration.php]: this should be a booleanNode (and please use the ``defaultFalse()`` method

Comment (Matrooskin) [inline DependencyInjection/Configuration.php]: I fixed that line. Btw. https://github.com/symfony/MonologBundle/blob/master/DependencyInjection/Configuration.php#L302 :D

Comment (stof) [inline DependencyInjection/Configuration.php]: the syslog ident is not a boolean

**Comment by Seldaek** [comment]
Thanks

**Pull request merged**